openerp/addons/base/ir/ir_qweb.py

```diff
@@ -344,7 +345,7 @@ def render_element(self, element, template_attributes, generated_attributes, qwe
                 for qwebcontext in (name, generated_attributes, inner, name)
             )
         else:
-            return "<%s%s/>" % (name, generated_attributes)
+            return u"<%s%s/>" % (name, generated_attributes)
```

Comment:
That's not correct, the current qweb implementation must only generate `str`. `name` should be encoded instead, as it is in the other branch of the conditional.